package/boot/uboot-envtools/files/mediatek_filogic

```diff
@@ -47,6 +47,7 @@ mercusys,mr90x-v1)
 netgear,wax220)
 	ubootenv_add_uci_config "/dev/mtd5" "0x0" "0x20000" "0x20000"
 	;;
+xiaomi,mi-router-wr30u-stock|\
 xiaomi,redmi-router-ax6000-stock)
```

Comment:
You have to handle `xiaomi,mi-router-wr30u-112m-nmbm` here too.

Reply:
Corrected. Thank you.